bochs/bx_debug/dbg_main.cc

```diff
@@ -2258,6 +2260,26 @@ void bx_dbg_disassemble_current(int which_cpu, int print_time)
   }
 }
 
+void bx_dbg_addlyt(const char* new_layoutpath)
```

Comment:
doesn't feel safe to me.
if new_path is too long - need to have a nice error message and not copy part of the path and hope for good

Reply:
oki! solved :D